packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2601,6 +2606,19 @@ export class ContainerRuntime
 
 		this.verifyNotClosed();
 
+		// Reference Sequence Number may be about to change, and it must be consistent across a batch, so flush now
+		this.outbox.flush();
```

Comment:
This feels fine, but historically we got burned by simplest changes in the flow. Should it be under kill-bit switch, just in case? Or that's not sufficient (as it's not just about flush here)?

Reply:
Right, I thought adding flush would be safe, I feel pretty confident that _this_ entry point will not be reentrant itself (it'll just be a new turn through the Inbound queue). I do have the kill-bit switch for the throwing, of course.

Reply:
Decided to put this behind the flag too - it actually made the config naming much more rational.